technology/overview.md

```diff
@@ -40,9 +40,11 @@ At the core of Pelagos’ value proposition is the programmable application logi
 
 By delivering authentic, timely, and verifiable data, the application logic layer powers rich, complex workflows that seamlessly interact across diverse blockchain ecosystems. This capability is essential for enabling advanced use cases like scalable Appchain execution, multichain liquidity management, and rapid Appchain launches &mdash; each reliant on accurate crosschain information and responsive contract behavior.
 
+<!-- can this stand as is without the restaking? -->
```

Comment:
@JekaMas again, if the security claims from basic pos are strong enough, then I am ok with leaving the below p as is if you are